engine/lib/phx/src/render/pixel_format.rs

```diff
@@ -1,39 +1,23 @@
-use super::gl;
-
-pub type PixelFormat = i32;
-
-#[no_mangle]
-pub static PixelFormat_Red: PixelFormat = gl::RED as PixelFormat;
-
-#[no_mangle]
-pub static PixelFormat_RG: PixelFormat = gl::RG as PixelFormat;
-
-#[no_mangle]
-pub static PixelFormat_RGB: PixelFormat = gl::RGB as PixelFormat;
-
-#[no_mangle]
-pub static PixelFormat_BGR: PixelFormat = gl::BGR as PixelFormat;
-
-#[no_mangle]
-pub static PixelFormat_RGBA: PixelFormat = gl::RGBA as PixelFormat;
-
-#[no_mangle]
-pub static PixelFormat_BGRA: PixelFormat = gl::BGRA as PixelFormat;
-
-#[no_mangle]
-pub static PixelFormat_Depth_Component: PixelFormat = gl::DEPTH_COMPONENT as PixelFormat;
+#[luajit_ffi_gen::luajit_ffi(with_impl = true)]
+#[derive(Debug, Clone, Copy)]
+pub enum PixelFormat {
+    DepthComponent = 0x1902, // gl::DEPTH_COMPONENT
```

Comment:
I think it might be better for us to instead have a `PixelFormatToGL` method, which uses a match to convert the enum values to GL constants (both here and across the other enums). That would help with the WGPU port as well, as we've decoupled those constants.

Reply:
I don't like these explicit numbers too and I already have an idea how to modify luajit-ffi so it supports non-numerical enum variants. Will implement it at the end of this refactoring.